src/transport/HttpClient.ts

```diff
@@ -22,6 +22,7 @@ type BoardNotePageContent = { elements: ElementDto[]; paths: PathDto[] };
 type BoardNotePageContentWithVersion = { content: BoardNotePageContent; version: number };
 type DocumentNotePageContentWithVersion = { content: string; version: number };
 export type PermissionDto = { user: User; access: string }
+export type RatingStats = { average: number; count: number; };
```

Comment:
Do we want to display number of ratings in this view?

Reply:
imo we do not need it, but choice is on you

Reply:
whatever tbh